velox/functions/prestosql/aggregates/tests/HistogramTest.cpp

```diff
@@ -245,10 +247,11 @@ TEST_F(HistogramTest, globalGroupByDate) {
 }
 
 TEST_F(HistogramTest, globalGroupByInterval) {
-  auto vector = makeFlatVector<IntervalDayTime>(
+  auto vector = makeFlatVector<int64_t>(
```

Comment:
why can't we keep using makeFlatVector() and other test utilities with UDTs?

Reply:
> why can't we keep using makeFlatVector() and other test utilities with UDTs?

That's a good question. We need a way to infer logical type (INTERVAL_DAY_TIME) from physical type (int64_t). Since mapping from physical to logical types is 1-to-many, going from physical to logical is not possible. 

That said, we do have C++ types defined for use in simple functions (IntervalDayType). Maybe we could use these type in makeFlatVector and similar utilities.